zone-setup/src/bin/zone-setup.rs

```diff
@@ -296,16 +296,13 @@ logdir /var/log/chrony
 log measurements statistics tracking
 
 # makestep <threshold> <limit>
-# We allow chrony to step the system clock if we are more than a day out,
-# regardless of how many clock updates have occurred since boot.
-# The boundary NTP servers are configured with local reference mode, which
-# means that if they start up without external connectivity, they will appear
-# as authoritative servers even if they are advertising January 1987
-# (which is the default system clock on a gimlet after boot).
-# This configuration allows a one-off adjustment once RSS begins and the
-# boundary servers are synchronised, after which the clock will advance
-# monotonically forwards.
-makestep 86400 -1
+# We allow chrony to step the system clock up to three times on startup if we
+# are more than a second out. Even though boundary NTP servers are configured
+# with local reference mode, they will not appear as authoritative servers
+# until they synchronize upstream. This configuration allows us to converge
+# quickly once the boundary servers converge. After this the clock will advance
+# monotonically forward.
+makestep 1.0 3
```

Comment:
I think this makes sense now that we have 3ebded84ae30e77062e7bef6cd90468714682e35. Prior to that, we had issues cold booting a rack with missing or delayed connectivity to the external NTP server. By the time it was available, chrony was past the three allowed updates and would refuse to step, leaving us on a very slow slew trajectory from 1987. It should be easy enough to replicate this scenario on a racklette and confirm.

Reply:
Is doing a sled reboot in a racklette sufficient to confirm this?

Reply:
I've performed the following test on a racklet.

1. Got Omicron fully up and running on this branch.
2. Took out the uplinks on both switches to take out upstream connectivity completely.
3. Power cycled the entire racklet.
4. Verified all the sleds came back up in 1986.
5. Waited 30 minutes.
6. Restored the uplinks.

With this procedure, all the sleds re-converged back to the current time in a timely manner and Omicron came back up.